### ISSUE TITLE
Adding less to css compilation on update

### ISSUE THREAD

**Issue by micker** [FLEXIcontent/flexicontent-cck#1010]
Hello when we use flexicontent update we lost less customisation, we need to edit one template and save to regenerate it
it can be cool to add less->css compilation on postinstall action

**Comment by ggppdk** [comment]
I think we don't need to change anything

If i understand you
The CORE LESS files and the LESS files of core templates must never be customized.

If someone needs a customization then the user can create a CSS override in its own CSS files

About less re-compile in your development website (if you have not done already)
You only need to turn on (auto) re-compile LESS in "Advanced" TAB of component options
(It costs very little as it check modification times but for live sites it does not need to be turned ON)

Then if you visit a backend and a frontend view the dependent CSS files will be auto-created
Thus your commits to GitHub will always contain the updated files

**Comment by micker** [comment]
But if user user less option in core template (no code juste option) the less files are updated  and customisation lost
We need to édit template template admin and save to regenerate good less customisation

**Comment by iamrobert** [comment]
J4 has shifted to Sass, and doesn't include a LESS compiler:

https://github.com/joomla/joomla-cms/issues/30569


- https://github.com/joomla/joomla-cms/tree/4.1-dev/administrator/templates/atum/scss
- https://github.com/joomla/joomla-cms/tree/4.1-dev/templates/cassiopeia/scss

**Comment by micker** [comment]
Yes maybe for next version WE will study it Can you help us to prépare bs5 update for all templates ? WE will unify HTML and HTML5 file and update all styles

**Comment by micker** [comment]
more explain
Before update
![image](https://user-images.githubusercontent.com/2539440/123556582-4486a480-d78c-11eb-99ec-6a108c2c41de.png)

After update
![image](https://user-images.githubusercontent.com/2539440/123556555-1ef99b00-d78c-11eb-9231-420e3c977985.png)

After resave template
![image](https://user-images.githubusercontent.com/2539440/123556584-45b7d180-d78c-11eb-8fe4-bc0c01e3c26a.png)

in a prod site, or maybe we need to check is less compilation is correctly configure for prod at first

**Comment by ggppdk** [comment]
We do not use the LESS compiler of Joomla 3
instead we have a new version of the LESS compiler

so the removal of it in J4 is not a problem
